design/src/rfcs/rfc0012_unsynchronized_versioning.md

```diff
@@ -0,0 +1,207 @@
+
+- __A: Crate has changed?__ The crate's source files and manifest will be hashed for the previous version
+  and the next version. If these hashes match, then the crate is considered unchanged.
+- __B: Minor bumped?__ The previous version is compared against the next version to see if the minor version
+  number was bumped.
+- __C: Version changed?__ The previous version is compared against the next version to see if it changed.
+- __D: Patch bumped?__ The previous version is compared against the next version to see if the patch version
+  number was bumped.
+- __E: Semverver passes?__ Runs [rust-semverver] against the old and new versions of the crate.
+  - If semverver fails to run (for example, if it needs to be updated to the latest nightly to succeed),
+    then fail CI saying that either semverver needs to be updated, or that a minor version bump is required.
+  - If semverver results in errors, fail CI indicating a minor version bump is required.
+  - If semverver passes, then pass CI.
+
+When running semverver, the path dependencies of the crate under examination should be updated to be crates.io
+references if there were no changes in those crates since the last public to crates.io. Otherwise, the types
+referenced from those crates in the public API will always result in breaking changes since, as far as the Rust
+compiler is concerned, they are different types originating from separate path-dependency crates.
+
+For CI, the `aws-sdk-rust/main` branch's `versions.toml` file is the source of truth for the previous release's
+crate versions and source code.
+
+### Yanking
+
+The publisher tool will be updated to read the `versions.toml` to yank all versions published in a release.
+This process will look as follows:
+1. Take a path to a local clone of the [aws-sdk-rust] repository
+2. Confirm the working tree is currently unmodified and on a release tag.
+3. Read `versions.toml` and print out summary of crates to yank
+4. Confirm with user before proceeding
+5. Yank crates
+
+### Changes Checklist
+
+- [ ] Update `rust-semverver` to a newer nightly that can compile `aws-smithy-client`
+- [ ] Establish initial `versions.toml` in `aws-sdk-rust/main`
+- [ ] Set version numbers in runtime crates in [smithy-rs]
+- [ ] Update the auto-sync tool to generate `versions.toml`
+- [ ] Create CI tool to check runtime crate version
+  - [ ] Integrate with `smithy-rs/main` CI
+  - [ ] Integrate with `aws-sdk-rust/main` CI
+- [ ] Update CI to verify no older runtime crates are used. For example, if `aws-smithy-client` is bumped to
+      `0.50.0`, then verify no crates (generated or runtime) depend on `0.49.0` or lower.
+
+**Estimate:** 2-4 dev weeks
+
+Phase 2: Stability and 1.x
+--------------------------
+
+When stabilizing to 1.x, the version process will stay the same, but with the introduction of the greater than zero
+major version number, the minor version bumps caused by version bumping runtime crates, updating models, or changing
```

Comment:
```suggestion
When stabilizing to 1.x, the version process will stay the same, but with the minor version bumps caused by version bumping runtime crates, updating models, or changing
```

The current wording feels a bit redundant but I don't feel super strongly about this